application/common/config/main.php

```diff
@@ -95,15 +96,39 @@
                     },
                 ],
                 [
-                    'class' => EmailTarget::class,
-                    'categories' => ['application'], // stick to messages from this app, not all of Yii's built-in messaging.
-                    'logVars' => [], // no need for default stuff: http://www.yiiframework.com/doc-2.0/yii-log-target.html#$logVars-detail
+                    'class' => EmailServiceTarget::class,
+                    'except' => [
+                        'yii\web\HttpException:400',
+                        'yii\web\HttpException:401',
+                        'yii\web\HttpException:404',
+                        'yii\web\HttpException:409',
+                        'Sil\EmailService\Client\EmailServiceClientException',
+                    ],
                     'levels' => ['error'],
+                    'logVars' => [], // Disable logging of _SERVER, _POST, etc.
                     'message' => [
-                        'from' => $mailerUsername,
                         'to' => $notificationEmail,
-                        'subject' => "ERROR - $idpName-id-broker [".YII_ENV."] Error",
+                        'subject' => 'ALERT - ' . $idpName . ' ID Broker [' . YII_ENV .']',
```

Comment:
Again, this was just an accidental carry-over from idp-pw-api's version. I'll change it back to "ERROR" here.